src/nodes.coffee

```diff
@@ -1945,7 +1945,7 @@ exports.Code = class Code extends Base
       # encountered, add these other parameters to the list to be output in
       # the function definition.
       else
-        if param.isComplex()
+        if param.isComplex() or param.isCall()
```

Comment:
So a call does not count as "complex"? What happens if we change `.isComplex()` to also check for calls instead?